packages/cli/src/commands/init/templates/app-react-email/ts/app/novu/emails/novu-onboarding-email.tsx

```diff
@@ -64,22 +66,22 @@ export const NovuWelcomeEmail = ({
               return (
                 <Section key={componentIndex}>
                   {component.type === "heading" ? (
-                    <Column>
-                      <h1 className={`text-${component.align}`}>
+                    <Section>
```

Comment:
using `Section` everywhere for consistency, it also fixes a React-email formatting issue where the `table` element breaks when using a mixture of `Column` and `Section` as component siblings.